bubble-wrap.yaml

```diff
@@ -2692,7 +2695,6 @@ layers:
         # building footprints, pre-extrusion
         footprints:
             filter:
-                $zoom: { max: 18 }
```

Comment:
The alternative would be to continue the pattern of area/volume thresholds for footprints and extrusions for higher zoom levels.